### ISSUE TITLE
Use name-that-color on existing color tags

### ISSUE THREAD

**Issue by ofermoshsaioff** [galex/name-that-color-intellij-plugin#11]
Great plugin! I'd love to have the ability to change existing color names into their "real" color names instead of the ones we currently use. Would be awesome!

**Comment by galex** [comment]
That's a great idea!

**Comment by fmatosqg** [comment]
Is this meant as a refactor thing? If it is then +1

Btw I love that page, so happy to have a plugin to automate my work :)

**Comment by galex** [comment]
@fmatosqg I think on this task I'd like to rename an existing color tag when the caret is on its name or on the color itself. Feel free to [open a new issue](https://github.com/galex/name-that-color-intellij-plugin/issues/new) and describe what you envision as refactor tool!

**Comment by fmatosqg** [comment]
After using a little bit I think that's exactly what I'd like it to do. Although I'd love it to pop up a confirmation dialog giving me the chance to edit the new name (I prefer camelCase). 

Another nice thing would be replacing the new name in all the places where it's used.

**Comment by galex** [comment]
Yes, if the plugin renames a color it should definitely change the references to it too.
